### PR TITLE
Fix duplicate H1 headers

### PR DESCRIPTION
**Pull request by szekelyzol**
Changes are for [this Asana task](https://app.asana.com/0/1205448597201625/1205546610661012).

**Summary**:

- Updated each affected page to only have one H1 header and bumped header levels accordingly
- Fixed typos (for example instanciation -> instantiation)
- Fixed table formatting for some SKDs for better readability
- Removed duplicate table of contents sections (Doctave generates a table of contents for each page, no need to define one in the page body)

### CODE REVIEW

**Review by olivierapivideo — APPROVED** [review]

File: templates/documentation/delivery-analytics/analytics.md [hunk 1]
```diff
@@ -84,7 +83,7 @@ api.video retains play event data for 30 days. If you select a time period that
 | `to`  | `date`  | `false`  | Use this optional query parameter to set the end date for the time period that you want analytics for.<br>  <br>- The API returns analytics data **excluding** the day you set in `to`.<br>- If you do not specify a `to` date, the API returns analytics data starting from the `from` date up until today, and **excluding** today.<br>- A valid date value is only **within the last 30 days**.<br>- The value you provide must follow the `YYYY-MM-DD` format.  |
 | `filter`  | `key-value pair`  | `false`  | Use this parameter to filter your results to a specific video or live stream in a project that you want analytics for.<br><br>You must use the `videoId:` or `liveStreamId:` prefix when specifying an identifier for a video or a live stream.<br><br>Note that the **Get play events for videos** endpoint only accepts video `ID` as a filter, and the **Get play events for live streams** endpoint only accepts live stream `ID` as a filter.  |
 
-### Dimension
+#### Dimension
```
Comment (arthemium) [inline templates/documentation/delivery-analytics/analytics.md:86]: @szekelyzol is this valid?

####

File: templates/documentation/sdks/player/apivideo-android-player-analytics.md [hunk 2]
```diff
@@ -94,49 +71,27 @@ Options instantiation is made with either mediaUrl or videoInfo.
...
+| `seek(from: Float, to: Float): Future<void>` | method to call when a seek event occurs, the `from` and `to` parameters are mandatory and should contain the seek start & end times in seconds |
+| `pause(eventTime: Float = currentTime): Future<void>` | method to call when the video is paused |
+| `destroy(eventTime: Float = currentTime): Future<void>` | method to call when the video player is disposed (eg. when the use closes the navigation tab) |
+| `currentTime` | field to call each time the playback time changes (it should be called often, the accuracy of the collected data depends on it) if you don't know event time. |
 
-#### `resume(eventTime: Float = currentTime): Future<void>`
-
-> method to call when the video playback is resumed after a pause
-
-#### `ready(eventTime: Float = currentTime): Future<void>`
```
Comment (arthemium) [inline templates/documentation/sdks/player/apivideo-android-player-analytics.md:113]: why did we remove this?

**Review by arthemium — APPROVED** [review]
LGTM, but we need to change the same things in the readme.md of each of the SDK

File: templates/documentation/sdks/player/apivideo-android-player-analytics.md (hunk 2 above)
Comment (szekelyzol) [reply templates/documentation/sdks/player/apivideo-android-player-analytics.md:113]: I only moved these to a table format, the content is still there.

File: templates/documentation/delivery-analytics/analytics.md (hunk 1 above)
Comment (szekelyzol) [reply templates/documentation/delivery-analytics/analytics.md:86]: Yep, it works!

**Pull request merged**